assets/common.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -e
```

Comment:
Is it a good idea to remove this? Generally we want to fail on errors.

Reply:
So I'm running helm upgrade in the background and then printing the logs AFTER helm exits
if helm exits on an error, the script dies which means no errors